modules/ROOT/pages/push/binding.adoc

```diff
@@ -1,8 +1,56 @@
 include::{partialsdir}/attributes.adoc[]
 :service-name: Push Notifications
 
-= Binding a {mobile-client} with the {service-name} service
+== Prerequisites
+
+[role="primary"]
+.Android
+
+****
+* [x] A Google account
+* [x] Access to the Firebase console. See link:google-setup.html[Firebase Setup] to get started.
+****
+
+[role="secondary"]
+.iOS
+
+****
+* [x] An Apple Developer account
+* [x] A provisioning certificate. Have a look at link:apple-setup.html[APNs Setup] for further instructions.
```

Comment:
Not sure if we should call it `provisioning certificate`. Maybe `APNs client TLS certificate` from [official doc](https://help.apple.com/developer-account/#/dev82a71386a).